xpra/client/client_base.py

```diff
@@ -204,7 +204,7 @@ def init_challenge_handlers(self, challenge_handlers):
                 items = (
                     "uri", "file", "env",
                     "kerberos", "gss",
-                    "u2f",
+                    "keycloak", "u2f",
```

Comment:
You define a `keycloak` challenge handler, but you are not including it any more?
How is this going to work?